cpp/src/gandiva/function_registry_common.h

```diff
@@ -199,6 +199,28 @@ typedef std::unordered_map<const FunctionSignature*, const NativeFunction*, KeyH
                  DataTypeVector{TYPE(), int64()}, int64(), kResultNullNever, \
                  ARROW_STRINGIFY(NAME##WithSeed_##TYPE))
 
+// HashSHA128 functions that :
+// - NULL handling is of type NULL_NEVER
+// - can return errors
+//
+// The function name includes the base name & input type name. gdv_fn_sha128_float64
+#define HASH_SHA128_NULL_NEVER(NAME, ALIASES, TYPE)                           \
```

Comment:
I think it is called SHA1 and not SHA128. The digest bit size is also 160 and not 128

Reply:
Already fixed.